pkg/boot/netboot/netboot.go

```diff
@@ -76,6 +76,12 @@ func getBootImage(schemes curl.Schemes, uri *url.URL, mac net.HardwareAddr, ip n
 		return nil, fmt.Errorf("failed to parse pxelinux config: %v", err)
 	}
 
+	if pc.DefaultEntry == "" {
+		pc.DefaultEntry = "LinuxImage"
+	}
 	label := pc.Entries[pc.DefaultEntry]
```

Comment:
to not panic here, do

label, ok := pc.Entires[pc.DefaultEntry]

right?

Reply:
yeah, I got confused for a second as I thought the blowup was in pxeboot.go, but I guess it was right here. Oops. Another round coming.
Right now DefaultEntry is "", where should it be set? The parser doesn't handle empty labels it seems.

Reply:
done